examples/remix-cms/app/routes/admin/articles.tsx

```diff
@@ -6,7 +6,9 @@ import { Article } from "~/models/Article";
 import { useChannel } from "~/utils/use-channel";
 
 export async function loader() {
-  const articles = await Article.with("user").orderBy("createdAt", "desc");
+  const articles = await Article.with("user")
+    .orderBy("createdAt", "desc")
+    .get();
```

Comment:
A limitation with ALS in CF Workers today is that custom thenables don't properly retain context: https://twitter.com/jasnell/status/1634764772121145344

Reply:
woohoo! https://twitter.com/jasnell/status/1761066798039044416